### ISSUE TITLE
[Bug]: Unknown Error - Code: X-5

### ISSUE THREAD

**Issue by GokayGonayYT** [ThioJoe/YT-Spammer-Purge#544]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I was using Sensitive-Smart-Mode but when it started scanning after a few seconds it gave the error

### Release version

YTSpammerPurge.2.15.0-Beta1

### Steps to reproduce

1. I Tried To Scan Recent Comments Across My Entire Channel
2. When It Asked For Maximum Comments I selected 1200
3. I selected Sensitive-Smart-Mode And Started It
4. After 2-3 Seconds It Gave The Error

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Traceback (most recent call last):canned: 43 | Matches Found So Far: 0
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1573, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1550, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1113, in primaryInstance
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YTSpammerPurge.py", line 1090, in scan_video
  File "Scripts\operations.py", line 134, in get_comments
TypeError: 'NoneType' object is not iterable
```

### Screenshots

![Bug Report](https://user-images.githubusercontent.com/55464905/152528003-e6669741-470f-4eea-b86d-be44de28a7af.png)

**Comment by ThioJoe** [comment]
Can you post a link to the channel

**Comment by GokayGonayYT** [comment]
> Can you post a link to the channel

https://www.youtube.com/channel/UC5Suyr5bkVZA8rF39Dix7Yw

**Comment by ThioJoe** [comment]
Hm yea it happens to me, definitely a bug

**Comment by ThioJoe** [comment]
Ok it's fixed with https://github.com/ThioJoe/YT-Spammer-Purge/commit/d08a2ddfc5dc290f4529f812333f3f33ac678ab6

It will be in the next release

**Comment by ThioJoe** [comment]
Ok new release is out with the fix, 2.15.0-Beta3
